third-party/libtm/CMakeLists.txt

```diff
@@ -11,12 +11,12 @@ set(INFRA_INCLUDE_DIR "${LIBTM_ROOT}/libtm/src/infra")
 set(LIBTM_SRC_DIR "${LIBTM_ROOT}/libtm/src")
 set(LIBTM_RESOURCES_DIR "${LIBTM_ROOT}/resources")
 
-include(versions.cmake)
+set(HOST_VERSION "0.19.3.1711")
```

Comment:
is this version maintained?

Reply:
I have no idea if this version number is used outside this project, but it is used to extract `LIBTM_VERSION_MAJOR`, etc., so I kept it for now.  I'd be happy to remove it.